### ISSUE TITLE
Вывод geojson в консоль

### ISSUE THREAD

**Issue by nicola-spb** [rendrom/rosreestr2coord#82]
Привет,
Добавьте, пожалуйста, параметр при выставлении которого результат будет выводиться в консоль (файл не будет создаваться). 

- при успешной операции - geoJson
- при ошибке - ошибка

Если есть большое количество кадастровых номеров и на каждый запрос создавать файл, читать из него geoJson, а потом его удалять, то будет убиваться жесткий диск. Удобнее было бы считывать с консоли результат и сохранять в нужном формате. А использовать библиотеку напрямую не получается т.к. пишу на другом языке программирования.

**Comment by nicola-spb** [comment]
Поизучал код, понял что при парсинге списка, сохраняется в csv (а не в отдельные файлы, как при еденичном поиске).  
Закрываю.

**Comment by rendrom** [comment]
Добрый день, скрипт изначально не рассчитан для обработки такого числа записей. Процесс записи результата в файлы всё-равно будет происходить для целей кэширования.

Вы можете использовать программный вызов Area с отключенным параметром кэширования use_cache=False вместо чтения вывода в консоль.

**Comment by nicola-spb** [comment]
Да, файлы все равно создаются, даже если запускать с параметром `-r` (без кеширования). 
Программный вызов не могу сделать т.к. пишу на другом языке. 

З.Ы. Я попробовал ограничить доступ на запись в папку geojson, но тогда все падает.
